src/Map.js

```diff
@@ -983,7 +995,7 @@ export const MAPCMap = ({ wrapperHeight = "100vh", mapFocus = "region", polyPoin
           {layers}
           <FeatureLayer
             url="https://geo.mapc.org/server/rest/services/transportation/landlines/FeatureServer/0"
-            key={featureQuery} //FORCE RELOAD ON QUERY CHANGE
+            key={`${featureQuery}-${featureLayerKey}`} //FORCE RELOAD ON QUERY CHANGE
```

Comment:
Instead of adding featureLayerKey as new state (to trigger the map to reload here), we can probably just reuse `showGaps`. If showGaps changes, or if the featureQuery changes, the key will update. Something like: 
```jsx
key={`${featureQuery}-${showGaps}`} 
```